tests/test_cli.py

```diff
@@ -94,6 +103,9 @@ def mock_network_requests():
         patch("mcstatus.server.JavaServer.ping", return_value=0), \
         patch("mcstatus.server.JavaServer.status", return_value=JavaStatusResponse.build(JAVA_RAW_RESPONSE)), \
```

Comment:
Oh, we forgot to drop 3.9 this year (in 3.10, they've added support for parentheses in the `with` block)